setup.py

```diff
@@ -247,27 +247,35 @@ def main():
 
     setup(
         name='bdsf',
-        version='1.9.2',
+        version='1.9.3a1',
```

Comment:
Please just make this 1.9.3 or 1.9.4.

Reply:
That's OK. I didn't want to do that until the code was at least reviewed.
You could even argue it should be `1.10` or `1.10.0`, because in the mean time we've already dropped support for Python 2.7, which is still supported by `1.9.2`.

Reply:
The version won't be actually increased until the pull request is merged. `1.10.0` is also fine with me, indeed dropping python 2.7 support is a major thing.

Reply:
Do we keep a ChangeLog somewhere? And if so, who's responsible for updating it?

Reply:
Yep, the changelog is at https://github.com/lofar-astron/PyBDSF/blob/master/bdsf/_version.py. I can update it. I'd also like to merge the other open PR (#165) before releasing a new version.

Reply:
Sorry, yes we do keep a changelog. In fact, two:
* bdsf/_version.py
* doc/source/whats_new.rst
We typically update this when a new release is actually released, but you could add changes earlier on.

Reply:
Ouch, so there are two places where the version number is set: in setup.py, and in bdsf/_version.py. In Rapthor setup.py fetches the version number from the _version.py file. Maybe that's something to consider here as well.

Reply:
OK, then it needs a bit more work 😄